my-app/src/frontend/pages/home/Home.js

```diff
@@ -1,12 +1,14 @@
 import { Filter } from "../../components/filter/Filter";
 import { Products } from "../../components/products/Products";
+import { Footer } from "../../components/footer/Footer";
```

Comment:
You can have one index.js in component and there export all the components, then here you can import all components in one line.